sa/SASCM.CLEO.ini

```diff
@@ -108,7 +102,13 @@
 0AF4=4,%4d% = read_string_from_ini_file %1s% section %2s% key %3s%
 0AF5=4,write_string %1s% to_ini_file %2s% section %3s% key %4s%
 
-; CLEO 4 FileSystemOperations.cleo
+; CLEO 5 FileSystemOperations.cleo
+0AAB=1,  does_file_exist %1s%
```

Comment:
Are white characters after comma intended?

Reply:
this is standard for conditional opcodes